modules/virt-about-cluster-checkup-framework.adoc

```diff
@@ -10,4 +10,13 @@ A _checkup_ is an automated test workload that allows you to verify if a specifi
 
 By using predefined checkups, cluster administrators can improve cluster maintainability, troubleshoot unexpected behavior, minimize errors, and save time. They can also review the results of the checkup and share them with experts for further analysis. Vendors can write and publish checkups for features or services that they provide and verify that their customer environments are configured correctly.
 
-Running a predefined checkup in the cluster involves setting up the namespace and service account for the framework, creating the `ClusterRole` and `ClusterRoleBinding` objects for the service account, enabling permissions for the checkup, and creating the input config map and the checkup job. You can run a checkup multiple times. 
+Running a predefined checkup in the cluster involves setting up the namespace and service account for the framework, creating the `ClusterRole` and `ClusterRoleBinding` objects for the service account, enabling permissions for the checkup, and creating the input config map and the checkup job. You can run a checkup multiple times.
+
+[IMPORTANT]
+====
+You must always:
+
+* Verify that the checkup image is from a trustworthy source before applying it.
+* Review the checkup permissions before creating the `ClusterRole` objects.
```

Comment:
Either "ClusterRoleBinding" should be added ("before creating the `ClusterRole` and `ClusterRoleBinding`objects"), or change "objects" to "object" (AFAIR only one `ClusterRole` object is needed for the job).

Reply:
In 4.11 the `checkup-framework` Job creates the `ClusterRoleBinding` objects automatically.
Users have to review the `ClusterRoleBinding` that they apply.
The phrasing is correct IMO.

Reply:
Very well, then LGTM.
Thanks for the clarification, Orel.